common/elasticsearch/src/main/scala/uk/ac/wellcome/elasticsearch/IndexConfig.scala

```diff
@@ -41,6 +41,36 @@ trait IndexConfigFields {
       textField("english").analyzer("english")
     )
 
+  def frenchTextField(name: String) =
+    textField(name).fields(
+      textField("french").analyzer("french")
+    )
```

Comment:
Possibly a daft question, but what are these functions for? I don't see you using them anywhere.

Reply:
They've been removed now...